.travis.yml

```diff
@@ -4,6 +4,9 @@ sudo: false
 language: java
 jdk: oraclejdk8
 
+install:
+  - echo "Override default Travis install step to avoid unnecessary './gradlew assemble'."
```

Comment:
Overriding?

Reply:
Yeah, that would read better in logs, rather than ask user to take some action